docs/source/conf.py

```diff
@@ -360,7 +362,8 @@ def _does_not_start_with_underscore(input_string):
             + "</a>"
         )
 
-        record = pd.DataFrame([modname, algorithm_type, author_info], index=COLNAMES).T
+        record_cells = [[modname, algorithm_type, author_info, maintainer_info]]
+        record = pd.DataFrame(data=record_cells, columns=COLNAMES)
         records += [record]
 
     df = pd.concat(records, ignore_index=True)
```

Comment:
I think I didn't explain myself correctly. This is what I meant.

```suggestion
        records.append([modname, algorithm_type, author_info, maintainer_info])

    df = pd.DataFrame(records, columns=COLNAMES)
```